compiler/java/src/main/java/com/c/lexer/Token.java

```diff
@@ -0,0 +1,31 @@
+package com.c.lexer;
+
+public class Token {
+    private TokenType type;
+    private String value;
+    private int lineNum;
+    private int colNum;
+
+    public Token(TokenType type, String value, int lineNum, int colNum) {
+        this.type = type;
+        this.value = value;
+        this.lineNum = lineNum;
+        this.colNum = colNum;
+    }
+
+    public TokenType getType() {
```

Comment:
Would be nice if you could add JavaDoc to all methods, not only to a few

Reply:
will do this once I'll implement the lexer in the next pull request